src/redisjson.rs

```diff
@@ -189,24 +189,26 @@ impl RedisJSON {
     }
 }
 
-#[allow(non_snake_case, unused)]
-pub unsafe extern "C" fn json_rdb_load(rdb: *mut raw::RedisModuleIO, encver: c_int) -> *mut c_void {
-    if encver < 2 {
-        panic!("Can't load old RedisJSON RDB"); // TODO add support for backward
+pub struct TypeMethods;
+
+impl TypeMethods {
```

Comment:
@gavrie isn't it better to make it a mod and not struct?

Reply:
Yes indeed; I'll do that.